crates/api_models/src/refunds.rs

```diff
@@ -61,6 +61,45 @@ pub struct RefundRequest {
     pub charges: Option<ChargeRefunds>,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Debug, ToSchema, Clone, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct RefundsCreateRequest {
+    /// The payment id against which refund is initiated
+    #[schema(
+        max_length = 30,
+        min_length = 30,
+        example = "pay_mbabizu24mvu3mela5njyhpit4",
+        value_type = String,
+    )]
+    pub payment_id: common_utils::id_type::GlobalPaymentId,
+
+    /// Unique Identifier for the Refund. This is to ensure idempotency for multiple partial refunds initiated against the same payment.
+    #[schema(
+        max_length = 30,
+        min_length = 30,
+        example = "ref_mbabizu24mvu3mela5njyhpit4",
+        value_type = String,
+    )]
+    pub merchant_reference_id: common_utils::id_type::RefundReferenceId,
```

Comment:
is this mandatory? We will create a refund id right if reference id is not passed.